lib/internal/Magento/Framework/Session/SessionManager.php

```diff
@@ -501,7 +516,29 @@ public function regenerateId()
             return $this;
         }
 
-        $this->isSessionExists() ? session_regenerate_id(true) : session_start();
+        // @codingStandardsIgnoreStart
```

Comment:
Why was this code fragment suppressed?

Reply:
It has been replaced with the if / else block till line 540